openhexa/toolbox/iaso/iaso.py

```diff
@@ -0,0 +1,124 @@
+import io
+import typing
+
+from openhexa.toolbox.iaso.api_client import ApiClient
+import polars as pl
+
+
+class IASO:
+    """
+    The IASO toolbox provides an interface to interact with the IASO.
+    """
+
+    def __init__(self, server_url: str, username: str, password: str) -> None:
+        """
+        Initializes the IASO toolbox.
+        :param client: The IASO toolbox client.
+
+        Examples:
+            >>> from openhexa.toolbox.iaso import IASO
+            >>> iaso = IASO(server_url="http://iaso-staging.bluesquare.org",
+            >>>             username="user",
+            >>>             password="pass")
+        """
+        self.api_client = ApiClient(server_url, username, password)
+
+    def get_projects(self, page: int = 0, limit: int = 10, **kwargs) -> dict:
+        """
+        Fetches projects list from IASO. Method is paginated by default. Pagination can be modified and additional
+        arguments can be passed as key value parameters.
+
+        Examples:
+            >>> from openhexa.toolbox.iaso import IASO
+            >>> iaso = IASO(client=ApiClient(url="http://iaso-staging.bluesquare.org",
+            >>>             username="user",
+            >>>             password="pass"))
+            >>> iaso.get_projects(page=1, limit=1, id=1)
+        """
+
+        params = kwargs
+        params.update({"page": page, "limit": limit})
+        response = self.api_client.request("GET", "/api/projects", params=params)
+        return response.json().get("projects")
+
+    def get_org_units(self, page: int = 0, limit: int = 10, **kwargs) -> dict:
+        """
+        Fetches org units from IASO. Method is paginated by default. Pagination can be modified and additional
+        arguments can be passed as key value parameters.
+
+        Examples:
+            >>> from openhexa.toolbox.iaso import IASO
+            >>> iaso = IASO(client=ApiClient(url="http://iaso-staging.bluesquare.org",
+            >>>             username="user",
+            >>>             password="pass"))
+            >>> projects = iaso.get_org_units(page=1, limit=1, id=1)
+        """
+        params = kwargs
+        params.update({"page": page, "limit": limit})
+        response = self.api_client.request("GET", "/api/orgunits", params=kwargs)
```

Comment:
```suggestion
        response = self.api_client.get("/api/orgunits", params=params)
```

Reply:
`params=kwargs` works since you just copy the reference into `params` and update the dictionary in-place (and thus the kwargs object) but I would prefer to update the kwargs directly OR using params here.